packages/components/buttons/primary-button/src/primary-button.styles.ts

```diff
@@ -6,12 +6,13 @@ import type { TPrimaryButtonProps } from './primary-button';
 const getSizeStyles = (size: TPrimaryButtonProps['size']) => {
   switch (size) {
     // rendering the same style for now to not introduce a breaking change
+    case '10':
     case 'small':
     case 'medium':
       return css`
         height: ${designTokens.heightForButtonAsMedium};
       `;
-
+    case '20':
     case 'big':
       return css`
         height: ${designTokens.heightForButtonAsBig};
```

Comment:
I believe we should not use the deprecated tokens in our code anymore and use the new ones instead.

Reply:
addressed [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-c10084319127c43b8f3365a94b946414de422e1b5df5272a1c28e43375b35e74L11-L18) @CarlosCortizasCT